src/Codacy/Coverage/Parser/CloverParser.php

```diff
@@ -64,7 +64,7 @@ private function makeFileReports(\SimpleXMLElement $node)
      */
     private function makeFileReportsFromFiles(\SimpleXMLElement $node, $fileReports)
     {
-        foreach ($node as $file) {
+        foreach ($node->file as $file) {
```

Comment:
yes, because it takes some other xml node that the one it should takes. Other part of the lib filter on file node, so with this I have no issue